public/react-services/group-handler.js

```diff
@@ -10,70 +10,91 @@
  * Find more information about this on the LICENSE file.
  */
 import { WzRequest } from './wz-request';
+import { UI_LOGGER_LEVELS } from '../../common/constants';
+import { UI_ERROR_SEVERITIES } from './error-orchestrator/types';
+import { getErrorOrchestrator } from './common-services';
 
 export class GroupHandler {
   static async removeGroup(group) {
     try {
-      const result = await WzRequest.apiReq(
-        'DELETE',
-        `/agents/groups/${group}`,
-        {}
-      );
+      const result = await WzRequest.apiReq('DELETE', `/agents/groups/${group}`, {});
       return result;
     } catch (error) {
-      return Promise.reject(error);
+      const options = {
+        context: `${GroupHandler.name}.removeGroup`,
+        level: UI_LOGGER_LEVELS.ERROR,
+        severity: UI_ERROR_SEVERITIES.BUSINESS,
+        store: true,
+        error: {
+          error: error,
+          message: error.message || error,
+          title: `Error removing selected group`,
+        },
+      };
+      getErrorOrchestrator().handleError(options);
```

Comment:
why is the service calling to ErrorHandling?  It should be throw to the upper layers (?)